release.py

```diff
@@ -204,20 +204,20 @@ def get_pref(prefs, name, request_fn):
 cmd("Verifying that you have no unstaged git changes", 'git diff --exit-code --quiet')
 cmd("Verifying that you have no staged git changes", 'git diff --cached --exit-code --quiet')
 
-release_version = raw_input("Release version (without any RC info, e.g. 0.10.2.0): ")
+release_version = raw_input("Release version (without any RC info, e.g. 1.0.0): ")
 try:
     release_version_parts = release_version.split('.')
-    if len(release_version_parts) != 4:
-        fail("Invalid release version, should have 4 version number components")
+    if len(release_version_parts) != 3:
```

Comment:
This is fine for everything past 1.0.0, but if we do want to make things releasable using this script on trunk (similar to how people use the merge script probably), then we'd want to maintain support for the 4-component version and validate ones starting with 0 vs >= 1.

I'm fine going either way (not maintaining that for simplicity and bug fixes on older branches will need to use the older release script, or just adding in a bit more logic here).

Reply:
I was thinking we'd just use the release script from the older branches for older releases.

Reply:
That's fine, might just need to be made clear on the release instructions wiki since it is different than how, e.g., the merge script is used.

Reply:
Will update the wiki accordingly.